examples/assets/css/main.css

```diff
@@ -567,4 +560,13 @@ code.tab3{
 }
 code.tab4{
 	margin-left:12px;
-}
\ No newline at end of file
+}
+
+.rdg-bottom-pinned-rows-sticky-wrapper .react-grid-checkbox-container {
```

Comment:
Is this css unique to the example?

Reply:
This is unique to the example. I think it's better to leave the customization flexibility to the user for whether to display the checkbox for total rows,